basics/test.ts

```diff
@@ -1,52 +1,76 @@
-import * as Kilt from "@kiltprotocol/sdk-js";
-import { Keyring } from "@kiltprotocol/utils";
-import { verifyDid } from "./addVerification2Did";
-import { generateAccounts } from "./generateAccount";
-import { generateDid } from "./generateDid";
-import { issueCredential } from "./issueCredential";
+import * as Kilt from '@kiltprotocol/sdk-js'
+import { verifyDid } from './addVerification2Did.ts'
+import { generateAccounts } from './generateAccount.ts'
+import { generateDid } from './generateDid.ts'
+import { Balances, KiltAddress, SignerInterface } from '@kiltprotocol/types'
+import { issueCredential } from './issueCredential.ts'
+import { claimW3N } from './claimW3N.ts'
 
 async function runAll(): Promise<void> {
-  let api = await Kilt.connect("wss://peregrine.kilt.io/");
-
-  console.log("connected");
-
-  const faucetAccount = Kilt.generateKeypair({
-    type: "ed25519",
-    seed: "0xe566520fec3ca23d80dfe9e9529ada463b93fc33f17219c1089de906f7253f1c",
-  });
-
-  // Yeni bir Keyring örneği oluştur
-  const keyring = new Keyring({ type: "ed25519" });
-
-  // Seed'den KeyringPair oluştur
-  const keyringPair = keyring.addFromSeed(
-    Buffer.from(
-      "e566520fec3ca23d80dfe9e9529ada463b93fc33f17219c1089de906f7253f1c",
-      "hex"
-    )
-  );
-
-  let { issuerAccount, submitterAccount, holderAccount, verifierAccount } =
-    generateAccounts();
-  console.log("Successfully transferred tokens");
-  submitterAccount = faucetAccount;
-  let issuerDid = await generateDid(faucetAccount, issuerAccount);
-  let holderDid = await generateDid(faucetAccount, holderAccount);
-  //let verifierDid = await generateVerifierDid(faucetAccount, verifierAccount)
+  let api = await Kilt.connect('wss://peregrine.kilt.io/')
+
+  console.log('connected')
+
+  const faucet = {
+    publicKey: new Uint8Array([
```

Comment:
Isn't there a better way than a bytes array? from seed for example?

Reply:
We do have access to the faucet mnemonic. 

I have switched it.